.github/workflows/scorecards.yml

```diff
@@ -0,0 +1,60 @@
+name: Scorecards supply-chain security
+on:
+  # Only the default branch is supported.
+  branch_protection_rule:
+  schedule:
+    - cron: '26 0 * * 0'
```

Comment:
What is the reason for running this workflow at specific times rather than just during the CI?

Reply:
Because new CVEs can come up at any time for the dependencies we are using, which isn't limited to when our CI runs or when we perform a new build.